test/e2e/common/pods.go

```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2ekubelet "k8s.io/kubernetes/test/e2e/framework/kubelet"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2ewebsocketutil "k8s.io/kubernetes/test/e2e/framework/websocket"
```

Comment:
It is better to set the alias name as `e2ewebsocket` for consistent name as the above.

Reply:
sure.

Reply:
done